edx_proctoring/templates/proctoring/proctored-exam-status.underscore

```diff
@@ -0,0 +1,13 @@
+<div class="exam-timer">
+<%- gettext("You are taking") %>
+<a href="<%- interpolate(gettext('%(exam_url_path)s'), { exam_url_path: exam_url_path }, true)%>" >
+
+<%- interpolate(gettext(' %(exam_display_name)s '), { exam_display_name: exam_display_name }, true) %>
+</a>
+<%- gettext(" exam as a proctored exam. Good Luck!") %>
```

Comment:
I don't suggestions have made it in yet. In summary, after today's walkthrough, I suggest:
1) Put quotation marks around the name of the exam.
2) Strike the occurrence of "exam" after the inserted exam name.
3) Replace "Good Luck!" Update the string so that it reads, ""You are taking {Exam_display_name} as a proctored exam. The timer on the right shows the amount of time remaining for you to complete the exam."
@griffresch @marcotuts @chrisndodge OK?